### PR TITLE
Firewatch 1.5

### PR DESCRIPTION
**Pull request by misterpantz**
#### Description

- Add more detailed docs on Firewatch initial release, with enough on risk score and rank to cover us until customization comes out.
- Combine the vulnerabilities and security posture subsections under security -- Firewatch has exposed how linked they are.

#### Related issue

none

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [x] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: docs/platform/security/vuln/vulnerabilities.md [hunk 1]
```diff
@@ -59,12 +59,15 @@ Risk factors are attributes that can elevate the risk that a CVE poses to your o
...
-| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed.             |
-| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates an operating system version that is no longer supported. |
+| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed on at least one asset with this CVE.             |
+| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates that at least one asset with this CVE is running an operating system version that is no longer supported. |
+| ![Database icon](/img/platform/security/db.png) | **Database** indicates that at least one asset with this CVE hosts a running database.              |
+| ![In use icon](/img/platform/security/use.png) | **In use** indicates that at least one asset with this CVE has a running service or is in active use. Examples are assets running SSHD, OpenSSH, NGINX, or Apache, or assets with open or listening ports. |
```
Comment (tas50) [inline docs/platform/security/vuln/vulnerabilities.md:69]: This is usually written sshd

File: docs/platform/security/vuln/vulnerabilities.md [hunk 2]
```diff
@@ -59,12 +59,15 @@ Risk factors are attributes that can elevate the risk that a CVE poses to your o
...
 
 | Icon                                          | Risk factor                                                                              |
 | --------------------------------------------- | ---------------------------------------------------------------------------------------- |
-| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed.             |
-| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates an operating system version that is no longer supported. |
+| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed on at least one asset with this CVE.             |
+| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates that at least one asset with this CVE is running an operating system version that is no longer supported. |
+| ![Database icon](/img/platform/security/db.png) | **Database** indicates that at least one asset with this CVE hosts a running database.              |
```
Comment (tas50) [inline docs/platform/security/vuln/vulnerabilities.md:68]: Can we list the database systems we currently detect here?

File: docs/platform/security/vuln/vulnerabilities.md [hunk 3]
```diff
@@ -59,12 +59,15 @@ Risk factors are attributes that can elevate the risk that a CVE poses to your o
...
 
-Mondoo also flags a CVE if the _assets_ that contain the CVE present their own risk factors:
+Mondoo also flags a CVE if the _assets_ that contain the CVE have factors that increase or decrease risk:
 
 | Icon                                          | Risk factor                                                                              |
 | --------------------------------------------- | ---------------------------------------------------------------------------------------- |
-| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed.             |
-| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates an operating system version that is no longer supported. |
+| ![Keys icon](/img/platform/security/keys.png) | **Accessible keys** indicates that key or credential information is exposed on at least one asset with this CVE.             |
+| ![EOL icon](/img/platform/security/eol.png)   | **End-of-life (EOL)** indicates that at least one asset with this CVE is running an operating system version that is no longer supported. |
```
Comment (tas50) [inline docs/platform/security/vuln/vulnerabilities.md:67]: Currently this actually means it's approaching or at EOl

**Review by tas50 — CHANGES_REQUESTED** [review]

**Comment by tas50** [comment]
It might be better for light mode to get the original SVGs or the risks vs using PNGs that have a background

**Comment by misterpantz** [comment]
> It might be better for light mode to get the original SVGs or the risks vs using PNGs that have a background

Done!

File: docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx [hunk 4]
```diff
@@ -142,47 +142,47 @@ To assign this role across all subscriptions under a management group, follow th
 
 1. Create the custom role definition.
 
-  Save this JSON content into a file named `mondoo-role.json`:
```
Comment (tas50) [inline docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx:145]: Was it intentional to update the indent here?

File: docs/platform/security/posture/advisories.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Not every known software issue has an advisory. Be sure to monitor vulnerabilities (CVEs) to find potential exploits in your infrastructure. To learn more, read [Find Vulnerabilities](/platform/security/posture/vulnerabilities).
+
+:::
+
+## Find advisories in your infrastructure
+
+Find advisories for assets in a space:
+
+1. In the [Mondoo Console](https://console.mondoo.com), [navigate to the space](/platform/start/navigate) in which you want to see vulnerabilities
```
Comment (tas50) [inline docs/platform/security/posture/advisories.md:21]: Should this have a space to match the others?

File: docs/platform/security/posture/advisories.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+   The list shows CVEs found in your infrastructure. For each CVE, Mondoo shows a rank (priority compared to other CVEs), score (CRITICAL, HIGH, MEDIUM, OR LOW), blast radius (calculated based on the affected assets), any [risk factors](#risk-factors), and when the vulnerability was first found in your infrastructure.
+
+   To learn how Mondoo calculates risk and rank, read [Security Findings](/platform/security/posture/findings/).
+
+3. To filter the list, enter text in the search bar. These are some examples:
+
+   - To show only advisories for a certain platform, enter all or part of the platform name. For example, type `windows`, `debian`, or `google`.
+
+   - To find an advisory related to a specific CVE number, enter all or part of the number. For example, type `2023-21755` or `1325`.
```
Comment (tas50) [inline docs/platform/security/posture/advisories.md:35]:
```suggestion
   - To find an advisory related to a specific CVE number, enter all or part of the number. For example, type `2024-21755` or `1325`.
```

File: docs/platform/security/posture/findings.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+title: Prioritize Security Findings
+sidebar_label: Prioritize Findings
+sidebar_position: 6
+description: Understand how Mondoo scores and ranks security findings and vulnerabilities
+image: /img/featured_img/mondoo-feature.jpg
+---
+
+Mondoo scores and ranks security findings to help you prioritize fixes to your infrastructure. Our unique algorithms calculates risk score, blast radius, and rank based on a combination of contextual risk factors in your environment. You can spend more time closing security gaps and less time sifting through irrelevant or low-priority findings.
```
Comment (tas50) [inline docs/platform/security/posture/findings.mdx:9]: should this be "calculate"?

File: docs/platform/security/posture/findings.mdx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+Mondoo assigns a risk score of Low, Medium, High, or Critical to each finding. The risk score is calculated using a base score and contextual risks.
+
+### Begin with a base score
+
+Each finding has a base score associated with it, which Mondoo uses as a foundation for calculating the risk score.
+
+For misconfigurations exposed by Mondoo's security policies, the base score comes from the policy. Each finding is the result of a _check_ in a policy. The base score is based on the _impact score_ of a check. For example, if an asset fails the check "Ensure / and /home are encrypted" in the Linux Workstation Security policy, that finding has a high base score because of its high impact score in the policy. To learn about policies and checks, read [Policy as Code](/platform/security/posture/pac/).
+
+For CVEs and advisories, the base score is the [CVSS score](/platform/security/posture/vulnerabilities/#cvss-score-and-metrics).
```
Comment (tas50) [inline docs/platform/security/posture/findings.mdx:31]: Do we want to spell out CVSS here similar to CVE above?

File: docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx (hunk 4 above)
Comment (misterpantz) [reply docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx:145]: Nope! Prettier. :(

File: docs/platform/security/posture/advisories.md (hunk 5 above)
Comment (misterpantz) [reply docs/platform/security/posture/advisories.md:21]: You mean a period? Cuz it needs a period!

File: docs/platform/security/posture/findings.mdx (hunk 7 above)
Comment (misterpantz) [reply docs/platform/security/posture/findings.mdx:9]: Yep! Thanks! Rewrote that sentence too many times.

File: docs/platform/security/posture/findings.mdx (hunk 8 above)
Comment (misterpantz) [reply docs/platform/security/posture/findings.mdx:31]: Ooh yeah.

File: docs/platform/security/posture/advisories.md (hunk 5 above)
Comment (tas50) [reply docs/platform/security/posture/advisories.md:21]: That's the word!

Comment (tas50) [inline docs/platform/security/posture/advisories.md:21]:
```suggestion
1. In the [Mondoo Console](https://console.mondoo.com), [navigate to the space](/platform/start/navigate) in which you want to see vulnerabilities.
```

File: docs/platform/security/posture/findings.mdx (hunk 7 above)
Comment (tas50) [inline docs/platform/security/posture/findings.mdx:9]:
```suggestion
Mondoo scores and ranks security findings to help you prioritize fixes to your infrastructure. Our unique algorithms calculate risk score, blast radius, and rank based on a combination of contextual risk factors in your environment. You can spend more time closing security gaps and less time sifting through irrelevant or low-priority findings.
```

File: docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx (hunk 4 above)
Comment (misterpantz) [reply docs/platform/infra/cloud/azure/azure-integration-scan-group.mdx:145]: I didn't mean to change this file at all.

**Review by tas50 — APPROVED** [review]

**Pull request merged**